### ISSUE TITLE
Cannot Install ISIS >8.0.3

### ISSUE THREAD

**Issue by astrostu** [USGS-Astrogeology/isis3#5523]
It [appears as though ISIS](https://github.com/DOI-USGS/ISIS3/blob/dev/CHANGELOG.md) versions 8.1.0 and 8.2.0 have been released, but so far I have been unable to install it.  Running `conda search -f isis` and `conda search -c usgs-astrogeology -f isis` only reveals up through version 8.0.3.  I thought this was a "me thing" with 8.1.0, but now that 8.2.0 appears to be out and I still can't update to any version >8.0.3, I wanted to post this.  Perhaps it is still a "me thing?"

**Comment by rbeyer** [comment]
The [Releases page](https://github.com/DOI-USGS/ISIS3/releases) indicates that 8.0.3 is the highest-versioned ISIS out (there was an 8.1.0-RC2, but no 8.1.0 itself).

I just installed a fresh ISIS via conda, and it was 8.0.3.

It is possible that the CHANGELOG is being prepared for a future 8.1.0 and 8.2.0 release, but it is odd that the CHANGELOG has specific dates, IN THE PAST, for those releases, but they do not show on the GitHub Releases page.

I suspect maybe it is a hiccup in the CHANGELOG and that those 8.1.0 and 8.2.0 aren't really released, but maybe something weirder is happening?  Would be nice to know.

**Comment by astrostu** [comment]
Ah!  Release vs changelog, that would be a "me thing" of assuming the changelog with – as you noted – dates in the past, including January of this year for 8.1.0, was an accurate indicator of what was released.  I think what also has tripped me up is that some of the bugs logged by USGS folks on here have stated they were using version 8.1 or even 8.2, so even though I understand they could have some internal RCs, combined with the changelog dates I thought that those versions might really be out in the wild.  Well, those AND combined with the fact that 8.2.0 is now out per the changelog, but I still couldn't see 8.1.0, it didn't make sense to me that 8.1.0 would just be an internal release.

Certainly, some clarification from (a) USGS folk(s) would be good on this.

**Comment by chkim-usgs** [comment]
Apologies for the confusion and delay, [8.1.0](https://github.com/DOI-USGS/ISIS3/discussions/5534) has just been released along with [8.2.0_RC2](https://github.com/DOI-USGS/ISIS3/discussions/5532). If there are no reported issues specific to 8.2.0_RC2 within the next month, 8.2.0 will be released.

**Comment by astrostu** [comment]
Thanks, @chkim-usgs.  Can you clarify for someone who's fairly ign'ant about this process the difference between the changelog and releases pages?  Are the dates supposed to coincide and this was an oversight, or are they really quite different things and I need to pay attention to the releases page instead?

**Comment by chkim-usgs** [comment]
Yes, our approach towards dating the changelog and creating the actual GitHub release has been a little loose and we are striving towards having both dates match. Most common reason the changelog date may be a few days different than the release date would be due to some issues during the release process which we hope to make more robust and efficient. 

The 8.1.0 release was more so an oversight as RCs are given about month before another RC or a stable release is cut.

**Comment by Kelvinrr** [comment]
Seem this is closable. Reopen if there are any more problems.

**Comment by astrostu** [comment]
I probably commented where I shouldn't with this, so I'll just re-open this issue: The changelog notes that version 9.0.0 was released 7 months ago, but it is not yet available per the releases page nor on conda.  Builds started today for LTS bug fixes to 9.0.0, so it would be nice if 9.0.0 could be made available, assuming that a plain-reading of those logs is correct that it was intended to be made available.

**Comment by acpaquette** [comment]
Hi @astrostu we are working on a 9.0.0 now and will hopefully have something by the end of the week. Apologies that the communication regarding any 9.0 releases has been sparse

**Comment by astrostu** [comment]
@acpaquette Thank you.  I see it was released yesterday or the day before.  Any idea how long it takes to show up in the conda channel?  Or, any explicit way to install it to force conda to look in a specific place?  Running "conda search -c usgs-astrogeology isis" still says the most recent version is 8.3.0.

**Comment by acpaquette** [comment]
@astrostu The linux release is out as we have an automatic pipeline release process for linux. Mac pipelines in AWS aren't as easy or cost effective so we do them manually, I am working on the Mac release and should have it up by the end of the day

**Comment by astrostu** [comment]
Whoops, didn't follow up.  Thank you, I've installed it on my Macs.  I know it seems like I might be (am?) a bit of a נודיען on this ... but, is there a chance that this process could be documented?  E.g., when I see builds for v9 and that an LTS version is being added but I don't see v9 available, I start to wonder if it's user error (me!) or something else.  Knowing that you might still be working on v9 that isn't released while then incorporating other bug fixes from previous versions, and knowing it takes a few days once something is released for it to propagate and some parts are manual would be possibly good to document.

**Comment by Kelvinrr** [comment]
@astrostu Apologies, I think there has been some confusion here that's mostly my fault. We recently changed the release schedule: https://github.com/DOI-USGS/ISIS3/discussions/5731 

I was also gone for two weeks when you made this post which caused more confusion in the dev team especially combined with me being on leave. We originally didn't plan to do a 9.0 for a long while because of how we changed it. 

But people seeing 9.0 stuff everywhere was causing a lot of confusion so we decided to move forward with 9.0RC3. NOT 9.0 full. We need another RC first since we had bugs fixes put in specific for 9.0 that needed to be addressed first.

The current docs are still in the process of being updated to match the new release schedule. TLDR what is happening rn: 

1. 9.0 RC3 is being released for more testing 
2. After a month if there are not issues, 9.0 gets released  

This will not include newer stuff like Mac ARM support and GDAL. For that we still recommend people install from the dev channel (`mamba install -c usgs-astrogeology/label/dev isis`). 

I think in the future we need a way to programmatically generate a page with what the latest versions of each software is and when an update is expected rather than a page that simply shows an example timeline.

**Comment by Kelvinrr** [comment]
9.0 Should go out of RC in early December, a lot of turbulence this last year.
